### PR TITLE
Better documentation

### PR DESCRIPTION
**Pull request by lemire**
This PR improves the README. Importantly, it illustrates what standard compliance means in this case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,26 @@ pip install can_ada
...
+urlstring = "https://www.GOoglé.com/./path/../path2/"
+import can_ada
+# prints www.xn--googl-fsa.com,
+# the correctly parsed domain name according to WHATWG
+print(can_ada.parse(urlstring).hostname)
+# prints /path2/
+# the correctly parsed pathname according to WHATWG
+print(can_ada.parse(urlstring).pathname)
+import urllib
+#prints www.googlé.com
```
Comment (TkTech) [inline README.md]: For my own sanity, can you add a space here and below so it's consistent ;)

File: README.md [hunk 2]
```diff
@@ -36,4 +56,17 @@ url.search = "?q=canada&safe=off"
 print(url) # https://google.com/search?q=canada&safe=off
 ```
 
+## Performance
+
+We find that `can_ada` maybe be faster than `urllib`:
```
Comment (TkTech) [inline README.md]: > We find that `can_ada` may be faster than `urllib`:

Comment (lemire) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (lemire) [reply README.md]: Done.

**Pull request merged**